src/psyclone/transformations.py

```diff
@@ -2840,101 +2863,89 @@ def name(self):
         '''
         return "ACCRoutineTrans"
 
-    def apply(self, kern, options=None):
+    def apply(self, node, options=None):
         '''
-        Modifies the AST of the supplied kernel so that it contains an
-        '!$acc routine' OpenACC directive.
-        This transformation affects the f2pygen and the PSyIR trees of
-        this kernel.
+        Add the '!$acc routine' OpenACC directive into the code of the
+        supplied kernel or routine.
```

Comment:
I think we need some words here to explain why we distinguish kernel and routine. Perhaps "... kernel (in a PSyKAl API such as GOcean or LFRic) or PSyIR Routine."

Reply:
Done